File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    for (int r = 0; r < h; r++)
    {
        for (int d = 0; d < h - 1 - r; d++)
        {
            printf(" ");
        }
        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int


def main():
    h = get_positive_int()
    for i in range(h):
        for j in range(h-i-1):
            print(" ", end="")
        for k in range(i+1):
            print("#", end="")
        print()


def get_positive_int():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            break
    return n


main()
